python/ray/_raylet.pyx

```diff
@@ -536,14 +536,14 @@ cdef execute_task(
                   b' "task_id": ' + task_id.Hex() + b'}')
 
     if <int>task_type == <int>TASK_TYPE_NORMAL_TASK:
-        title = "ray_worker:{}()".format(function_name)
-        next_title = "ray_worker"
+        title = "ray_worker::{}()".format(function_name)
+        next_title = "ray_worker::IDLE"
         function_executor = execution_info.function
     else:
         actor = worker.actors[core_worker.get_actor_id()]
         class_name = actor.__class__.__name__
-        title = "ray_{}:{}()".format(class_name, function_name)
-        next_title = "ray_{}".format(class_name)
+        title = "ray_actor::{}.{}()".format(class_name, function_name)
+        next_title = "ray_actor::{}".format(class_name)
```

Comment:
This unfortunately makes it too long to see in `top` most of the time. Can we keep it as just `ray_`?

Reply:
Consider `ray::actor.method()`